dropwizard-metrics-graphite/src/main/java/io/dropwizard/metrics/graphite/GraphiteReporterFactory.java

```diff
@@ -51,6 +58,9 @@
     @NotNull
     private String prefix = "";
 
+    @NotNull
```

Comment:
Should we put a `OneOf` annotation on this only allowing the values of "tcp" or "udp"?